packages/mdc-floating-label/README.md

```diff
@@ -2,43 +2,50 @@
 title: "Floating Label"
 layout: detail
 section: components
-excerpt: "An animated text caption for a text field or select."
+excerpt: "An animated text caption for a Text Field or Select."
 path: /catalog/input-controls/floating-label/
 -->
 
 # Floating Label
 
-Floating labels display the type of input a field requires. Every text field and select should have a label. Labels are aligned with the input line and always visible. They can be resting (when a field is inactive and empty) or floating. The label is a text caption or description for the text field.
+Floating labels display the type of input a field requires. Every Text Field and Select should have a label. Labels are aligned with the input line and always visible. They can be resting (when a field is inactive and empty) or floating. The label is a text caption or description for the Text Field.
```

Comment:
@kfranqueiro Should I add an exception for full-width variant?

`Every Text Field and Select should have a label, except full-width variant should use a placeholder`

Reply:
Good call...

```
except for full-width text fields, which use the input's `placeholder` attribute instead.
```